package.json

```diff
@@ -18,39 +18,44 @@
     "dist"
   ],
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^11.0.2",
+    "@jsenv/github-pull-request-filesize-impact": "^2.6.0",
+    "@rollup/plugin-json": "^4.0.2",
     "@rollup/plugin-node-resolve": "^7.1.1",
     "@rollup/plugin-replace": "^2.3.1",
-    "@jsenv/github-pull-request-filesize-impact": "^2.6.0",
+    "@zeit/ncc": "^0.22.1",
     "bluebird": "^3.7.2",
     "concurrently": "^5.1.0",
     "construct-style-sheets-polyfill": "^2.3.5",
     "cross-env": "^7.0.2",
     "esm": "^3.2.25",
+    "make-fetch-happen": "^8.0.4",
     "mocha": "^7.1.1",
-    "node-fetch": "^2.6.0",
     "opn": "^6.0.0",
     "rimraf": "^3.0.2",
     "rollup": "^2.2.0",
     "rollup-plugin-terser": "^5.3.0",
+    "source-map-support": "^0.5.16",
     "terser": "^4.6.7",
     "whatwg-fetch": "^3.0.0"
   },
   "scripts": {
-    "build": "rimraf dist && rollup -c",
-    "build:dev": "rollup -c --environment dev",
+    "build": "rimraf dist && concurrently -n w: 'yarn:build:*'",
+    "build:node": "node build-node.js",
+    "build:browser:prod": "rollup -c",
+    "build:browser:dev": "rollup -c --environment dev",
     "footprint": "npm run footprint:systemjs && npm run footprint:sjs",
     "footprint:systemjs": "cat dist/system.min.js | gzip -9f | wc -c",
     "footprint:sjs": "cat dist/s.min.js | gzip -9f | wc -c",
     "test": "concurrently -n w: 'npm:test:*'",
-    "test:internals": "mocha -b -r esm --file test/mocha-setup.js test/import-map.js test/system-core.js test/url-resolution.js",
```

Comment:
The `esm` package throws errors in node 13 for me.

Reply:
Nice to remove this!